dvc/repo/experiments/queue/local.py

```diff
@@ -124,22 +125,31 @@ def worker(self) -> "TemporaryWorker":
         )
 
     def spawn_worker(self):
-        from shortuuid import uuid
-
         from dvc_task.proc.process import ManagedProcess
 
         logger.debug("Spawning exp queue worker")
         wdir_hash = hashlib.sha256(self.wdir.encode("utf-8")).hexdigest()[:6]
-        node_name = f"dvc-exp-{wdir_hash}-1@localhost"
+        number = 1
+        node_name = f"dvc-exp-{wdir_hash}-{number}@localhost"
+        worker_status = self.active_worker()
+        while node_name in worker_status:
+            number += 1
```

Comment:
We don't want to give each worker a new number. The `number` field should only be changing when we explicitly use `exp queue start --jobs <number>` (which is still disabled to always be 1).